### ISSUE TITLE
Kategoriezuordnung wird nicht mehr gleich angezeigt

### ISSUE THREAD

**Issue by heikohaensel** [pecuniabanking/pecunia-client#410]
Ich habe heute die Version 1.2 installiert. Wenn nicht nun in der Übersicht Umsätze per Drag & Drop auf eine Kategorie ziehe, wird sie nicht mehr sofort in der Spalte der Umsatzzeile angezeigt.
Erst nach dem ich links auf ein anderes Konto wechsle und wieder zurück, wird die Kategorie in der Spalte angezeigt.

**Comment by Alicia33** [comment]
Das kann ich für 10.9.5 nur bestätigen.

**Comment by heikohaensel** [comment]
Mike, diesen Bug kann ich im aktuellen Master immer noch nachvollziehen. Ich werde das mal analysieren und versuchen zu fixen.

**Comment by mike-lischke** [comment]
Ja, gut, das nervt mich auch. Allerdings habe ich diesen Part gerade komplett umgebaut. Also solltest du das im ui-rework branch fixen. Hab den gerade zu Github gepusht.

**Comment by heikohaensel** [comment]
Mike, kannst Du mir einen Tip geben, wie ich den neuen Branch auch in meinen Fork des Projektes bekomme?

Danke

**Comment by mike-lischke** [comment]
Das steht irgendwo in den FAQs von Github. Du legst dir zwei remotes an: für das originale Pecunia Repo und deinen Fork (für deinen Fork hast du das ja sicher schon). Dann kannst du statt von deinem Fork vom Original pull und mergen. Dann pusht du das zu deinem Fork auf Github und kannst auch zugleich den ui-rework branch auschecken und darin arbeiten.

**Comment by heikohaensel** [comment]
Danke, ich schau dann mal. Ich dachte das muss man direkt auf Github machen, da die anderen Branches auch in meinem Fork des Projektes auftauchen.
